### PR TITLE
Add references to other ddd-crew repos for deeper insights

### PR DESCRIPTION
**Pull request by yellowbrickc**
Proposal for #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,8 @@ As a starting point, we recommend [The Business Model Canvas](https://www.strate
 
 ![The Business Model Canvas](resources/business-model-canvas.png)
 
+Available templates can be found in [this repository](https://github.com/ddd-crew/virtual-modelling-templates)
```
Comment (NTCoding) [inline README.md:119]: We don't provide a template for the Business Model Canvas. We use the one from Strategyzr https://www.strategyzer.com/canvas. Miro also provides a BMC template as well as some other tools.

I think there are 2 choices here:

1. Add a link to Strategyzr.com

2. Don't add a link and trust people will see the link Strategyzr.com in the image itself

File: README.md [hunk 2]
```diff
@@ -153,6 +155,8 @@ As a starting point, we recommend [EventStorming](https://www.eventstorming.com/
 
 ![EventStorming](resources/event_storming.jpeg)
 
+More details regarding EventStorming are available in [this repository](https://github.com/ddd-crew/eventstorming-glossary-cheat-sheet)
```
Comment (NTCoding) [inline README.md:158]: There's a link to EventStorming right before the image and one 4 lines below, it feels like adding another link here is redundant.

If the goal is to add a link to the DDD Crew EventStorming repo I would change one/both of the existing links.

File: README.md [hunk 3]
```diff
@@ -183,6 +187,8 @@ As a starting point, we recommend carving up your event storm into sub-domains a
 
 ![Sub-domains on an EventStorm](resources/event_storm_sub_domains.png)*Credit: Alberto Brandolini*
 
+More details and templates about context maps can be found in [this repository](https://github.com/ddd-crew/context-mapping)
```
Comment (NTCoding) [inline README.md:190]: I think this link duplicates the link to context mapping above the image. I would simply replace the link above the image to point to this one if its our main reference for context mapping.

If it's just a link to another resource then I would put this link in the tools section below.

File: README.md [hunk 4]
```diff
@@ -260,6 +266,7 @@ We can optimise how people collaborate with each other if we align teams with co
 As a starting point, we recommend visualising sociotechnical architecture with the [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps). A brief overview of the most important patterns can be found under the [context-mapping](https://github.com/ddd-crew/context-mapping) GitHub Project.
 
 ![Context Map](resources/context-map-cheat-sheet.png)*Credit: Michael Plöd*
+More details and templates about context maps can be found in [this repository](https://github.com/ddd-crew/context-mapping)
```
Comment (NTCoding) [inline README.md:269]: This link appears to be a duplicate of the context mapping link above the image.

File: README.md (hunk 1 above)
Comment (yellowbrickc) [reply README.md:119]: 👍

Comment (yellowbrickc) [reply README.md:119]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (yellowbrickc) [reply README.md:158]:
```suggestion
```

**Comment by yellowbrickc** [comment]
I just close it then, it is too much hassle for not much revenue :)

**Pull request closed without merging**